homework/12_V/11_Georgi_Stanachkov/homework1/main.py

```diff
@@ -0,0 +1,28 @@
+import math
+import re
+
+
+def find_relative_position(x1, y1, r1, x2, y2, r2):
+    centers_distance = math.sqrt((x1 - x2) * (x1 - x2) + (y1 - y2) * (y1 - y2))
+
+    if centers_distance == r1 + r2:
+        print("Circle A touches circle B in one mutual point on their borders")

```

Comment:
The prints should either be removed (if you added those for debug) or turned into a debug log.